### ISSUE TITLE
Update for pandoc 2.0

### ISSUE THREAD

**Issue by mclearc** [joostkremers/pandoc-mode#71]
Are there any plans in the works for updating pandoc-mode to work with pandoc 2.0? Right now they don't seem to be compatible.

**Comment by joostkremers** [comment]
Yes, I will definitely be updating `pandoc-mode` to work with pandoc 2.0. Unfortunately, I currently don't have the time to look into it. Might take a few weeks.

**Comment by joostkremers** [comment]
I haven't made a full update to Pandoc 2 yet, but at least the new options have now been added. That will hopefully get you going.

If you run into trouble, let me know.

**Comment by mclearc** [comment]
Thanks - much improved over the few hacks I made. md-latex/pdf is working well.

**Comment by joostkremers** [comment]
Great. I hope to integrate the new input and output formats soon, too. It just requires rethinking the hydra menus, because there are now officially too many output formats to fit in the alphabet. :smile:
